releases/adm/sciencebeam-alert-rules.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: sciencebeam.rules
+  namespace: adm
```

Comment:
should this be the namespace sciencebeam is deployed in? i.e. `prod`
or should this file even be moved to `prod`? Or perhaps included in `sciencebeam-texture.yaml`?

Reply:
Currently Prometheus is configured to only look for PrometheusRules in its own namespace: https://github.com/prometheus-community/helm-charts/blob/f551c35d120cc74b774a46b8160904af7bc57135/charts/kube-prometheus-stack/values.yaml#L1794

This is the first time we have an application/product of our own specifying PrometheusRule. So this question didn't really come up for me so far.

I currently have no preference. There is an argument to be made to have the PrometheusRule live with the rest of the application's yaml. On the other hand this rule actually relies on the metrics provided by a different application (ingress-nginx), so it could justifiably live in the `monitoring` / `adm` namespace.

I leave it up to you. This is the test cluster, so feel free to experiment.